hybridse/src/udf/default_udf_library.cc

```diff
@@ -945,6 +945,36 @@ void DefaultUdfLibrary::InitStringUdf() {
 
                 @since 0.4.0
         )r");
+    RegisterExternal("regexp_like")
+        .args<StringRef, StringRef, StringRef>(reinterpret_cast<void*>(
+            static_cast<void (*)(StringRef*, StringRef*, StringRef*, bool*, bool*)>(
+                udf::v1::regexp_like)))
+        .return_by_arg(true)
+        .returns<Nullable<bool>>()
+        .doc(R"r(
+                @brief pattern match same as REGEXP_LIKE predicate
+
+                @param target: string to match
+
+                @param regular expression: the regular expression match pattern 
```

Comment:
briefly summary the regex standard

Reply:
added brief summary in Rules section with reference to https://www.ibm.com/docs/en/watson-explorer/11.0.1?topic=queries-posix-regular-expression-syntax-examples.